File: SMarketPlace/SMarketPlace.Web/Modules/ServerTypes/JobMP/JobSucursalesForm.ts
import { IntegerEditor, StringEditor, DateEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface JobSucursalesForm {
    Idintegrador: IntegerEditor;
    NombreMp: StringEditor;
    Descripcion: StringEditor;
    Dias: StringEditor;
    Hora: StringEditor;
    Activo: IntegerEditor;
    FechaInicio: DateEditor;
    Diario: IntegerEditor;
}

export class JobSucursalesForm extends PrefixedContext {
    static formKey = 'JobMP.JobSucursales';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!JobSucursalesForm.init)  {
            JobSucursalesForm.init = true;

            var w0 = IntegerEditor;
            var w1 = StringEditor;
            var w2 = DateEditor;

            initFormType(JobSucursalesForm, [
                'Idintegrador', w0,
                'NombreMp', w1,
                'Descripcion', w1,
                'Dias', w1,
                'Hora', w1,
                'Activo', w0,
                'FechaInicio', w2,
                'Diario', w0
            ]);
        }
    }
}

File: SMarketPlace/SMarketPlace.Web/Modules/JobMP/JobSucursales/JobSucursalesForm.cs
using Serenity.ComponentModel;
using System;

namespace SMarketPlace.JobMP.Forms;

[FormScript("JobMP.JobSucursales")]
[BasedOnRow(typeof(JobSucursalesRow), CheckNames = true)]
public class JobSucursalesForm
{
    public int Idintegrador { get; set; }
    public string NombreMp { get; set; }
    public string Descripcion { get; set; }
    public string Dias { get; set; }
    public string Hora { get; set; }
    public int Activo { get; set; }
    public DateTime FechaInicio { get; set; }
    public int Diario { get; set; }
}